### PR TITLE
chore: Master sync

### PR DESCRIPTION
**Pull request by hjetpoluru**
Its a master sync PR to update the change log and package version against develop

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25040?quickstart=1)

### CODE REVIEW

**Review by benjisclowder — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Re-enable the opt-in modal (2df1eb566b)
+
+## [11.16.4]
+### Changed
+- Temporarily disable the opt-in modal ([#25007](https://github.com/MetaMask/metamask-extension/pull/25007))
+
 ## [11.16.3]
 ### Changed
-- Update Trezor logo ([#24343]https://github.com/MetaMask/metamask-extension/pull/24343)
+- Update Trezor logo ([#24343](https://github.com/MetaMask/metamask-extension/pull/24343))
```
Comment (DDDDDanica) [inline CHANGELOG.md:19]: 💯

**Review by DDDDDanica — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (hjetpoluru) [reply CHANGELOG.md:19]: @DDDDDanica , thanks will be updated in the next master-sync which will happen today.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.16.7]
+### Fixed
+- Fix bug that breaks dapps that expect users to switch chains manually([#25046]https://github.com/MetaMask/metamask-extension/pull/25046)
+
+## [11.16.6]
+### Added
+- Add a Basic Functionality Toggle to settings, enabling users to opt-out of some features that send network requests to external services  ([#23456](https://github.com/MetaMask/metamask-extension/pull/23456))
```
Comment (DDDDDanica) [inline CHANGELOG.md:15]:
```suggestion
- Add a Basic Functionality Toggle to settings, enabling users to opt-out of some features that send network requests to external services ([#23456](https://github.com/MetaMask/metamask-extension/pull/23456))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,9 +6,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.16.7]
+### Fixed
+- Fix bug that breaks dapps that expect users to switch chains manually([#25046]https://github.com/MetaMask/metamask-extension/pull/25046)
+
+## [11.16.6]
```
Comment (DDDDDanica) [inline CHANGELOG.md:13]: I could be understanding it wrongly, since we never released 11.16.6, should we move changelog to 11.16.7?

Comment (danjm) [reply CHANGELOG.md:13]: No, I think we should retroactively create a 11.16.6 releases page. The release does exist, even if we didn't release it

File: CHANGELOG.md (hunk 2 above)
Comment (danjm) [reply CHANGELOG.md:15]: usually we don't modify the changes in master sync PRs, as they are meant to get develop matching what is on master. We could make a chance like this in a follow up PR against develop

**Review by danjm — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (DDDDDanica) [reply CHANGELOG.md:15]: Clear ~

**Review by DDDDDanica — APPROVED** [review]

**Review by benjisclowder — APPROVED** [review]

**Pull request merged**